app/models/merit/point_rules.rb

```diff
@@ -0,0 +1,30 @@
+# Be sure to restart your server when you modify this file.
+#
+# Points are a simple integer value which are given to "meritable" resources
+# according to rules in +app/models/merit/point_rules.rb+. They are given on
+# actions-triggered, either to the action user or to the method (or array of
+# methods) defined in the +:to+ option.
+#
+# 'score' method may accept a block which evaluates to boolean
+# (recieves the object as parameter)
+
+module Merit
+  class PointRules
+    include Merit::PointRulesMethods
+
+    def initialize
+      # score 10, :on => 'users#create' do |user|
+      #   user.bio.present?
+      # end
+      #
+      # score 15, :on => 'reviews#create', :to => [:reviewer, :reviewed]
+      #
+      # score 20, :on => [
+      #   'comments#create',
+      #   'photos#create'
+      # ]
+      #
+      # score -10, :on => 'comments#destroy'
```

Comment:
If we go with my comment above about creating a new object to hold the task history, you could use this method to add demerits.